test/integration/targets/pids/tasks/main.yml

```diff
@@ -0,0 +1,59 @@
+# Test code for the pid module
+# Copyright 2018, Saranya sridharan
+
+- name: "Installing PidOf"
+  yum:
+    name: sysvinit-tools
+    state: present
+  when: ansible_distribution == 'CentOS'
+  
+- name: "Installing setproctitle"
+  pip:
+    name: setproctitle
+    version: 1.1.10
+
+- name: "Checking empty result" 
+  pids: 
+    name: "blahblah"
+  register: emptypids
+  
+  
+- name: "Verify the Process IDs (PIDs) returned is empty"
+  assert:
+    that:
+    - emptypids is not changed
+    - emptypids.pids == [] 
+
+  
+- name: Copying the python script
```

Comment:
Run the script directly from its original location instead of making a copy to execute.